config/routes.rb

```diff
@@ -1,3 +1,17 @@
 Rails.application.routes.draw do
+  resources :posts
+  
+  get 'posts/index'
+  get 'posts/show'
+  get 'posts/new'
+  get 'posts/edit'
+  post 'posts/new', to: 'post#create'
+  
+  get 'users/index'
```

Comment:
You can substitute lines 10 and 11 here for `resources :users, only: [:index, :show]`. Try it out. If you see a route error, see `$ rails routes | grep user` and make sure to replace all reference of `users_index_path` to `users_path` because that should be the path statement to index by convention.

[`OPTIONAL`]

Reply:
Changed